ProjectManagement.BusinessLogic/Exceptions/GuardExtensions.cs

```diff
@@ -0,0 +1,38 @@
+﻿
+//using Microsoft.eShopWeb.ApplicationCore.Exceptions;
+using ProjectManagement.BusinessLogic.Exceptions;
+using ProjectManagement.Domain.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Ardalis.GuardClauses
```

Comment:
Please, name your namespaces the same as a path in a file system. In this case it should be `ProjectManagement.BusinessLogic.Exceptions`